### ISSUE TITLE
Cannot refer the alias with variable

### ISSUE THREAD

**Issue by IzekChen** [GoshPosh/grafana-meta-queries#85]
I am try to query elasticsearch then using metaqueries to do the timeshift.
In grafana you can use variable inside alias
[https://grafana.com/docs/grafana/latest/features/datasources/elasticsearch/#series-naming-and-alias-patterns](series-naming-and-alias-patterns)

Would it be possible to use variable in the alias, then refer it in the meta query "Metric" column

![image](https://user-images.githubusercontent.com/7790230/70218949-f27fc780-177e-11ea-8865-b473c93fba2c.png)

**Comment by Gauravshah** [comment]
@IzekChen you should be able to refer them as `OG` and `test`

**Comment by IzekChen** [comment]
Is that means I cannot use variable in the metric field?
Need to set it up one by one, right?

**Comment by Gauravshah** [comment]
yup, group bys aren't supported yet. there was some progress https://github.com/GoshPosh/grafana-meta-queries/issues/14 but not completed
